### PR TITLE
[example] Add a example of add one marker symbol to map

### PR DESCRIPTION
**Pull request by Chaoba**
PRs must be submitted under the terms of our Contributor License Agreement [CLA](https://github.com/mapbox/mapbox-maps-android/blob/main/CONTRIBUTING.md#contributor-license-agreement).
Fixes: < Link to related issues that will be fixed by this pull request, if they exist >

## Pull request checklist:
 - [x] Briefly describe the changes in this PR.
 - [x] Include before/after visuals or gifs if this PR includes visual changes.
 - [ ] Write tests for all new functionality. If tests were not written, please explain why.
 - [ ] Add example if relevant.
 - [ ] Document any changes to public APIs.
 - [ ] Apply changelog label ('breaking change', 'bug :beetle:', 'build', 'docs', 'feature :green_apple:', 'performance :zap:', 'testing :100:') or use the label 'skip changelog'
 - [x] Add an entry inside this element for inclusion in the `mapbox-maps-android` changelog: `<changelog>[example] Add a example of add one marker symbol to map</changelog>`.

### Summary of changes
This PR add a new example which add a single marker with symbol layer and aligns with the one in gl js https://docs.mapbox.com/mapbox-gl-js/example/add-a-marker/
image

### CODE REVIEW

File: app/src/main/java/com/mapbox/maps/testapp/examples/AddOneMarkerSymbolActivity.kt [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+import kotlinx.android.synthetic.main.activity_add_marker_symbol.*
+
+/**
+ * Example showing how to add a marker on map with symbol layer
+ */
+class AddOneMarkerSymbolActivity : AppCompatActivity() {
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_add_marker_symbol)
```
Comment (kiryldz) [inline app/src/main/java/com/mapbox/maps/testapp/examples/AddOneMarkerSymbolActivity.kt:24]: nit: if we're still doing `jumpTo` below - we could use `activity_simple_map.xml` here.

File: app/src/main/java/com/mapbox/maps/testapp/examples/AddOneMarkerSymbolActivity.kt [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+    mapView.getMapboxMap().also {
+      it.jumpTo(
+        CameraOptions.Builder()
+          .center(Point.fromLngLat(12.550343, 55.665957))
```
Comment (kiryldz) [inline app/src/main/java/com/mapbox/maps/testapp/examples/AddOneMarkerSymbolActivity.kt:29]: point to const

File: app/src/main/java/com/mapbox/maps/testapp/examples/AddOneMarkerSymbolActivity.kt [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+          .build()
+      )
+    }.loadStyle(
+      styleExtension = style(Style.MAPBOX_STREETS) {
+        // prepare blue marker from resources
+        +image(BLUE_ICON_ID) {
+          bitmap(BitmapFactory.decodeResource(resources, R.drawable.blue_marker_view))
+        }
+        +geoJsonSource(SOURCE_ID) {
+          geometry(Point.fromLngLat(12.550343, 55.665957))
```
Comment (kiryldz) [inline app/src/main/java/com/mapbox/maps/testapp/examples/AddOneMarkerSymbolActivity.kt:40]: Same as above - to const.

File: app/src/main/res/values/example_descriptions.xml [hunk 4]
```diff
@@ -51,6 +51,7 @@
     <string name="description_styles_dds_geojson_circle_layer_clusters_description">Use GeoJSON and circle layers to visualize point data in clusters.</string>
     <string name="description_change_icon_size">Symbol layer icon size change</string>
     <string name="description_add_marker_symbol">Add simple marker symbols</string>
+    <string name="description_add_one_marker_symbol">Add a marker symbol</string>
```
Comment (kiryldz) [inline app/src/main/res/values/example_descriptions.xml:54]: Is there a similar structure of name and description on iOS?
If yes - could we also align example name and descroption?

**Comment by kiryldz** [comment]
@Chaoba - why skip changelog? imho could add changelog entry `[examples] Add a example of add one marker symbol to map`.

File: app/src/main/res/values/example_descriptions.xml (hunk 4 above)
Comment (Chaoba) [reply app/src/main/res/values/example_descriptions.xml:54]: Good point.

**Review by pengdev — APPROVED** [review]

**Pull request merged**